test/unit/player.test.js

```diff
@@ -164,13 +166,16 @@ QUnit.test('should get current sources from source tag', function(assert) {
   const tag = document.getElementById('example_1');
   const player = TestHelpers.makePlayer({}, tag);
 
+  this.clock.tick(1);
```

Comment:
I got these to work because the source cache is now updated when the source is selected. We can't actually know what the currentSource will be until then. I am not sure if we want to cache the sources when we get them, then update the cache when a source is actually set.